torchvision/utils.py

```diff
@@ -121,10 +126,60 @@ def save_image(
             If a file object was used instead of a filename, this parameter should always be used.
         **kwargs: Other arguments are documented in ``make_grid``.
     """
-    from PIL import Image
     grid = make_grid(tensor, nrow=nrow, padding=padding, pad_value=pad_value,
                      normalize=normalize, range=range, scale_each=scale_each)
     # Add 0.5 after unnormalizing to [0, 255] to round to nearest integer
     ndarr = grid.mul(255).add_(0.5).clamp_(0, 255).permute(1, 2, 0).to('cpu', torch.uint8).numpy()
     im = Image.fromarray(ndarr)
     im.save(fp, format=format)
+
+
+def draw_bounding_boxes(
+    image: torch.Tensor,
+    boxes: torch.Tensor,
+    colors: Optional[List[str]] = None,
+    labels: Optional[List[str]] = None,
+    width: int = 1,
+    font: Optional[ImageFont] = None
```

Comment:
Given that we won't be using this function in torchscript, I'm ok having the input type of the function to be PIL-specific

Reply:
I'm not terribly excited about this TBH:

- On one hand the method receives a uint8 tensor as input (not a PIL image) and hides completely any dependency on PIL. I would agree with earlier comments of yours that it's a bit odd that we expose ImageFont here.
- On the other hand, using PIL's ImageFont gives the flexibility to the user to do whatever they want without having to deal on our side with the details on how to instantiate the object. It's surely is ugly though and makes for a weird API.

I could try to create a `font` parameter similar to PIL with description "[A filename or file-like object containing a TrueType font.](https://pillow.readthedocs.io/en/stable/reference/ImageFont.html)" and a `font_size`. Thoughts?

Reply:
I implemented the above changes on the latest commit. Let me know if you prefer the ImageFont vs passing font path and size.

Reply:
Have a look on the latest commit for an alternative to passing ImageFont. We can choose any of the two options, I'm OK with both.